resources/options.js

```diff
@@ -19,20 +28,18 @@ function changeListener() {
 function saveOptions() {
     chrome.storage.sync.set({
         // General
-        loggingEnabled: $('#logging').prop('checked'),
+        loggingEnabled: $(optionsID(0)).prop('checked'),
         // Privacy
-        trackTabs: $('#tabs').prop('checked'),
-        trackComments:  $('#comments').prop('checked'),
-        trackPeerComments: $('#peerComments').prop('checked'),
-        trackFocus: $('#focus').prop('checked'),
+        trackTabs: $(optionsID(1)).prop('checked'),
+        trackComments:  $(optionsID(2)).prop('checked'),
+        trackPeerComments: $(optionsID(3)).prop('checked'),
+        trackFocus: $(optionsID(4)).prop('checked'),
         // Security
-        hashUsername: $('#username').prop('checked'),
-        hashRepo: $('#repo').prop('checked'),
-        hashFile: $('#file').prop('checked')
+        hashUsername: $(optionsID(5)).prop('checked'),
+        hashRepo: $(optionsID(6)).prop('checked'),
+        hashFile: $(optionsID(7)).prop('checked')
         // Hints
-    }, function() {
-        Materialize.toast("Options saved!", 2000);
-    });
+    }, function() {});
```

Comment:
I think this object that is passed to the store can also be optimalized with a for-loop. This would require some renaming however. Would become something like:
```
function saveOptions() {
    var saveObject = {};
    for (i = 0..7) {
        saveObject[options[i]] = $(optionsId(i)).prop('checked');
    }
    chrome.storage.sync.set(saveObject, function() {});
}
```